src/views/User/__tests__/UsersTable.test.js

```diff
@@ -0,0 +1,60 @@
+import UsersTable from '../UsersTable.vue';
+import { shallowMount } from '@vue/test-utils';
+
+// Fixture
+import users from '@src/__fixtures__/users.js';
+
+describe('UsersTable.vue', () => {
+    const firstUserName = users[0].name;
+    const secondUserName = users[1].name;
+
+    let mocks;
+    let propsData;
+    let wrapper;
+
+    beforeEach(() => {
+        mocks = { $router: [] };
+        propsData = { users };
+        wrapper = shallowMount(UsersTable, { mocks, propsData });
+    });
+
+    test('should call the viewTokens method when view tokens link on action dropdown menu is clicked', () => {
+        const spy = jest.spyOn(wrapper.vm, 'viewTokens');
+
+        wrapper.find('button.actions').trigger('click');
+        wrapper.find('a.tokens').trigger('click');
+
+        expect(spy).toHaveBeenCalled();
+    });
+
+    test('should sort users by username when "User Name" header is clicked', () => {
+        wrapper.find('.table-header-filter').trigger('click');
+
+        expect(wrapper.find('.username').text()).toEqual(secondUserName);
+    });
+
+    test('should reverse sort users by username when "User Name" header is clicked', () => {
+        wrapper.find('.table-header-filter').trigger('click');
+        wrapper.find('.table-header-filter').trigger('click');
```

Comment:
Add a comment or adjust the test name to explain why we're double clicking here.